Xamarin.Forms.Platform.iOS/Material/MaterialSliderRenderer.cs

```diff
@@ -0,0 +1,183 @@
+			if (height == 0)
+				height = nfloat.IsInfinity(size.Height) ? 12 : size.Height;
+
+			return new CGSize(12, height);
+		}
+
+		protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+		{
+			base.OnElementPropertyChanged(sender, e);
+
+			if (e.PropertyName == Slider.MaximumProperty.PropertyName)
+				UpdateMaximum();
+			else if (e.PropertyName == Slider.MinimumProperty.PropertyName)
+				UpdateMinimum();
+			else if (e.PropertyName == Slider.ValueProperty.PropertyName)
+				UpdateValue();
+			else if (e.PropertyName == Slider.MinimumTrackColorProperty.PropertyName)
+				UpdateMinimumTrackColor();
+			else if (e.PropertyName == Slider.MaximumTrackColorProperty.PropertyName)
+				UpdateMaximumTrackColor();
+			else if (e.PropertyName == Slider.ThumbColorProperty.PropertyName)
+				UpdateThumbColor();
+		}
+
+		void UpdateMaximum()
+		{
+			Control.MaximumValue = (nfloat)Element.Maximum;
+		}
+
+		void UpdateMinimum()
+		{
+			Control.MinimumValue = (nfloat)Element.Minimum;
+		}
+
+		void UpdateValue()
+		{
+			nfloat value = (nfloat)Element.Value;
+			if (value != Control.Value)
+				Control.Value = value;
+		}
+
+		void UpdateSliderColors()
+		{
+			UpdateMinimumTrackColor();
+			UpdateMaximumTrackColor();
+			UpdateThumbColor();
+		}
+
+		void UpdateMinimumTrackColor()
+		{
+			Color color = Element.MinimumTrackColor;
+			if (color.IsDefault && _defaultMinimumTrackColor == null)
+				return;
+
+			if (_defaultMinimumTrackColor == null)
+				_defaultMinimumTrackColor = Control.GetTrackFillColor(UIControlState.Normal);
+
+			if (color.IsDefault)
+				Control.SetTrackFillColor(_defaultMinimumTrackColor, UIControlState.Normal);
+			else
+				Control.SetTrackFillColor(color.ToUIColor(), UIControlState.Normal);
+		}
+
+		void UpdateMaximumTrackColor()
+		{
+			Color color = Element.MaximumTrackColor;
+			if (color.IsDefault && _defaultMaximumTrackColor == null)
+				return;
+
+			if (_defaultMaximumTrackColor == null)
+				_defaultMaximumTrackColor = Control.GetTrackBackgroundColor(UIControlState.Normal);
+
+			if (color.IsDefault)
+				Control.SetTrackBackgroundColor(_defaultMaximumTrackColor, UIControlState.Normal);
+			else
+				Control.SetTrackBackgroundColor(color.ToUIColor(), UIControlState.Normal);
+		}
+
+		void UpdateThumbColor()
+		{
+			Color color = Element.ThumbColor;
+			if (color.IsDefault && _defaultThumbColor == null)
+				return;
+
+			if (_defaultThumbColor == null)
+				_defaultThumbColor = Control.GetThumbColor(UIControlState.Normal);
+
+			if (color.IsDefault)
+				Control.SetThumbColor(_defaultThumbColor, UIControlState.Normal);
+			else
+				Control.SetThumbColor(color.ToUIColor(), UIControlState.Normal);
+		}
+
+		void OnControlValueChanged(object sender, EventArgs eventArgs)
+		{
+			Element.SetValueFromRenderer(Slider.ValueProperty, Control.Value);
+		}
+
+		// TODO: decide if we want to just by default like Android does
+		//[Export("slider:shouldJumpToValue:")]
```

Comment:
The native iOS renderer has a touch gesture detector to handle taps. Done via the platform-specifics. For Material, this is already implemented by default. However, we may want to DISABLE this feature.